suite-common/fiat-services/src/limiter.ts

```diff
@@ -1,22 +1,26 @@
 import { differenceInMilliseconds } from 'date-fns';
 
+import { scheduleAction } from '@trezor/utils';
+
 export class RateLimiter {
     // Poor man's rate limiter
     // Slows down requests so there is `delayMs` gap between them
     private delayMs: number; // gap between each request
+    private timeoutMs: number; // timeout
```

Comment:
comment `// timeout` for property named `timeoutMs` seems really unnecessary here